common/route.h

```diff
@@ -18,13 +18,15 @@ struct gossmap_node;
  * @direction: 0 (dest node_id < src node_id), 1 (dest node_id > src).
  * @node_id: the node_id of the destination of this hop.
  * @amount: amount to send through this hop.
+ * @total_amount: The amount the channel was funded with.
```

Comment:
I would actually just use htlc_max, but if you want this, please call it "capacity"?

Reply:
No, this needs to be the overall capacity (yes, I'll rename it), since the HTLC limit is, as the name suggests, per HTLC, and we are running a tally of how much we've already assigned to the channel.